Tests/StoreKitUnitTests/PurchasesOrchestratorTests.swift

```diff
@@ -9,246 +9,55 @@
-                                                  storeMessagesHelper: self.mockStoreMessagesHelper)
-        self.storeKit1Wrapper.delegate = self.orchestrator
-    }
-
-    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-    func setUpOrchestrator(
-        storeKit2TransactionListener: StoreKit2TransactionListenerType,
-        storeKit2StorefrontListener: StoreKit2StorefrontListener
-    ) {
-        self.orchestrator = PurchasesOrchestrator(productsManager: self.productsManager,
-                                                  paymentQueueWrapper: self.paymentQueueWrapper,
-                                                  systemInfo: self.systemInfo,
-                                                  subscriberAttributes: self.attribution,
-                                                  operationDispatcher: self.operationDispatcher,
-                                                  receiptFetcher: self.receiptFetcher,
-                                                  receiptParser: self.receiptParser,
-                                                  transactionFetcher: self.mockTransactionFetcher,
-                                                  customerInfoManager: self.customerInfoManager,
-                                                  backend: self.backend,
-                                                  transactionPoster: self.transactionPoster,
-                                                  currentUserProvider: self.currentUserProvider,
-                                                  transactionsManager: self.transactionsManager,
-                                                  deviceCache: self.deviceCache,
-                                                  offeringsManager: self.mockOfferingsManager,
-                                                  manageSubscriptionsHelper: self.mockManageSubsHelper,
-                                                  beginRefundRequestHelper: self.mockBeginRefundRequestHelper,
-                                                  storeKit2TransactionListener: storeKit2TransactionListener,
-                                                  storeKit2StorefrontListener: storeKit2StorefrontListener,
-                                                  storeMessagesHelper: self.mockStoreMessagesHelper)
-        self.storeKit1Wrapper.delegate = self.orchestrator
-    }
-
-    var transactionPoster: TransactionPoster {
-        return .init(
-            productsManager: self.productsManager,
-            receiptFetcher: self.receiptFetcher,
-            transactionFetcher: self.mockTransactionFetcher,
-            backend: self.backend,
-            paymentQueueWrapper: self.paymentQueueWrapper,
-            systemInfo: self.systemInfo,
-            operationDispatcher: self.operationDispatcher
-        )
-    }
+//  Created by Mark Villacampa on 20/2/24.
 
-}
+protocol PurchasesOrchestratorTests {
```

Comment:
Again this file appears to have many changes but it's really just been renamed to `BasePurchasesOrchestratorTests.swift` and then re-used for the `PurchasesOrchestratorTests` protocol